src/libsync/clientsideencryption.cpp

```diff
@@ -61,6 +61,31 @@ namespace {
     const char e2e_cert[] = "_e2e-certificate";
     const char e2e_private[] = "_e2e-private";
     const char e2e_mnemonic[] = "_e2e-mnemonic";
+
+    QList<QByteArray> oldCipherFormatSplit(const QByteArray &cipher)
+    {
+        const auto separator = QByteArrayLiteral("fA=="); // BASE64 encoded '|'
```

Comment:
Just to be sure: This will always take a concatenated string of separately Base64 encoded strings ending with `fA==`, right? (Since having `|` somewhere in between would be encoded differently.)

Reply:
Yes.